src/app/_components/common/Notifications.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import React, { useState } from "react";
+import NotificationsIcon from "../icons/notifications";
+import { Popover, PopoverTrigger, PopoverContent } from "./Popover";
+import GiftIcon from "../icons/gift";
+
+const notifications = [
+  {
+    icon: "gift",
+    message: "Soledad Martínez canjeó su código!",
+  },
+  {
+    icon: "gift",
+    message: "Rosina Cataldo canjeó su código!",
+  },
+];
+const Notifications: React.FC = () => {
```

Comment:
We should add a blank line here between the array and the function definition
```suggestion
const notifications = [
  {
    icon: "gift",
    message: "Soledad Martínez canjeó su código!",
  },
  {
    icon: "gift",
    message: "Rosina Cataldo canjeó su código!",
  },
];

const Notifications: React.FC = () => {
```